### PR TITLE
added new file gitnew.md to create your own adventure

### PR DESCRIPTION
**Pull request by kranthig3012**
Please review this, @casey-collab

### CODE REVIEW

File: english/gitnew/gitnew.md [hunk 1]
```diff
@@ -0,0 +1 @@
+click[here](www.google.com) to open google website
```
Comment (casey-collab) [inline english/gitnew/gitnew.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

Comment (casey-collab) [inline english/gitnew/gitnew.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/gitnew/gitnew.md (hunk 1 above)
Comment (kranthig3012) [inline english/gitnew/gitnew.md]: Please review this, @casey-collab

**Pull request closed without merging**